esmvaltool/cmorizers/data/formatters/datasets/esacci_biomass.py

```diff
@@ -0,0 +1,174 @@
+"""ESMValTool CMORizer for ESACCI-BIOMASS above-ground biomass (agb) data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    ftp://anon-ftp.ceda.ac.uk/neodc/esacci/biomass/data/agb/maps
+
+Last access
+    20250716
+
+Download and processing instructions
+    Download 10 km file:
+      v6.0/netcd/ESACCI-BIOMASS-L4-AGB-MERGED-10000m-fv6.0.nc
+    Put file in ${RAWOBS}/Tier2/ESACCI-BIOMASS
+"""
+
+import datetime
+import glob
+import logging
+import os
+from copy import deepcopy
+
+import iris
+import numpy as np
+from dask import array as da
+from esmvalcore.cmor.table import CMOR_TABLES
+from esmvalcore.preprocessor import extract_time
+
+from esmvaltool.cmorizers.data.utilities import (
+    flip_dim_coord,
+    save_variable,
+    set_global_atts,
+)
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
```suggestion
```

This should not be necessary, logging is configured [here](https://github.com/ESMValGroup/ESMValTool/blob/e977fce4cf8a741fb7ae1ea572995bd3a257a1bf/esmvaltool/cmorizers/data/cmorizer.py#L87-L89).